lib/lotus/view/rendering/partial_finder.rb

```diff
@@ -34,7 +35,9 @@ class PartialFinder < TemplateFinder
         # @since 0.4.3
         # @api private
         def find
-          if path = partial_template_under_view_path
+          if cached_template = find_cached_template
```

Comment:
@stevehook Why do we need this double tier finding? I assume that the load process finds all the partials and put them in the appropriate registry (`Configuration#partials`).

At this point no further modifications are allowed. All the partials should be in that registry. In other words, we want to eager load everything, without falling back to runtime finding of partials.